sign.js

```diff
@@ -219,6 +219,9 @@ var signAsync = module.exports.signAsync = function (opts) {
       var promise
       if (opts.identity) {
         debuglog('`identity` passed in arguments.')
+        if (opts.skipIdentityValidation) {
```

Comment:
Control question is: whether to option-name? Whether to validate identity? It is why originally name skipIdentityValidation was suggested. If we inverse and simplify — whether to validate identity. 

Electron-builder has option npmRebuild, but npm here is a prefix. forceCodeSigning another example.